Makefile

```diff
@@ -0,0 +1,27 @@
+MSBUILD = xbuild
+MDOC = mdoc/bin/Release/mdoc.exe
+
+all: prepare build
```

Comment:
`prepare` should *not* be an `all` dependency.

The rationale is that if I update a submodule in my local tree *without* a corresponding commit -- because I'm still writing code for that commit! -- `make all` *should not* **reset** the submodule I updated. That gets *really* annoying to deal with.

Java.Interop and xamarin-android have `make prepare` as separate targets, not part of `all`.